helloworld-mutual-ssl-secured/configure-certs.cli

```diff
@@ -0,0 +1,19 @@
+#configure a key-store in the Elytron subsystem. The path to the keystore file doesn’t actually have to exist yet.
```

Comment:
Minor but there could be a space after each # and we should be consistent about using lower case or upper case at the beginning. Since this is a sentence, you could use # Configure ...

Reply:
Fixed!